cockroachcloud/configure-scim-provisioning.md

```diff
@@ -0,0 +1,144 @@
+   1. Contact your account team to enable your {{ site.data.products.db }} organization in SCIM limited access.
+
+1. If your IdP is Okta, SCIM provisioning can be enabled only on a [custom SAML authentication method](configure-cloud-org-sso.html#saml). This requirement is imposed by Okta, and is not part of the SCIM or SAML protocols.
+
+Individual IdPs may impose different requirements, and the exact steps and requirements for enabling SCIM autoprovisioning depend upon your IdP. Refer to your IdP's documentation about configuring SCIM provisioning.
+
+## Configure SCIM provisioning on Okta
+
+The exact steps and requirements for enabling SCIM provisioning depend upon your IdP. At a minimum, you must provide your IdP two pieces of information:
+
+- The endpoint to the {{ site.data.products.db }} SCIM API, `https://cockroachlabs.cloud/api/scim/v2`.
+- The API token of a {{ site.data.products.db }} service account with the [**Org Administrator (legacy)** role](authorization.html#org-administrator-legacy).
+
+To add SCIM provisioning to a SAML application in Okta:
+
+1. Log in to Okta Admin Dashboard as an admin user.
+1. Click **Applications** and edit the [SAML](configure-cloud-org-sso.html#saml) application for your {{ site.data.products.db }} organization.
+1. Click **Edit**.
+1. Click **Provisioning**.
+1. Select **SCIM** and click **Save**.
+1. In the integration's settings page, click **Provisioning** again, then click **Edit**.
+1. Click **Integrations**. This tab controls the app integration's authentication to the {{ site.data.products.db }} API. Set:
+
+    <ul><li><b>SCIM connector base URL</b>: <code>https://cockroachlabs.cloud/api/scim/v2</code></li><li><b>API authentication token</b>: the API token for a {{ site.data.products.db }} <a href="managing-access.html#create-a-service-account">service account</a> with the <a href="authorization.html#org-administrator-legacy"><b>Org Administrator (legacy)</b> role</a></li><li><b>Unique identifier field for users</b>: <code>userName</code></li><li><b>Authentication Mode</b>: <b>HTTP Header</b></li></ul>
+
+1. Click **Test Connector Configuration**.
+1. Click **Save**.
+1. Click **To App**. This tab controls assignment of Okta identities to {{ site.data.products.db }}. To allow provisioning and deprovisioning of users, ensure that **Create Users** and **Deactivate Users** are selected, and make any other desired changes.
+1. Optionally, click **To Okta**. This tab allows you to perform a one-time import of a {{ site.data.products.db }} organization's existing users into Okta. Refer to Okta's documentation about mapping individual fields. Make any desired changes.
+
+To learn more, refer to [Add SCIM Provisioning to App Integrations](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SCIM.htm) in the Okta documentation
+
+## Assign a user or group
+
+To provision an IAM user or group to {{ site.data.products.db }}, you assign the user or group to the app integration.
```

Comment:
replace every instance where you have "IAM user" with just "user" or "person", and same for groups, but don't do that for admins so keep where you have "IAM admin"